cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -52,7 +52,7 @@
 _SYC_GATE_FAMILY = cirq.GateFamily(ops.SYC)
 _SQRT_ISWAP_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP)
 _SQRT_ISWAP_INV_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP_INV)
-_CZ_GATE_FAMILY = cirq.GateFamily(cirq.CZ)
```

Comment:
since you are just add `CZPowGate cz_pow_gate`, it is better to add another CZ_POWER_GATE_FAMILY instead of modify the existing CZ ones